features/nanostack/mbed-mesh-api/source/WisunInterface.cpp

```diff
@@ -542,6 +544,60 @@ mesh_error_t WisunInterface::read_mac_statistics(mesh_mac_statistics_t *statisti
     return ret_val;
 }
 
+mesh_error_t WisunInterface::info_get(ws_rpl_info_t *info_ptr)
+{
+    if (info_ptr == NULL) {
+        return MESH_ERROR_PARAM;
+    }
+
+    rpl_dodag_info_t dodag_ptr = {0};
+    uint8_t global_address[16] = {0};
+    uint8_t rpl_instance_count;
+    uint8_t instance_id_list[10];
+    uint8_t instance_id = RPL_INSTANCE_LOCAL;
+    uint8_t instance_id_new;
+    uint8_t instance_index;
+    rpl_instance_count = rpl_instance_list_read(&instance_id_list[0], sizeof(instance_id_list));
+
+    if (rpl_instance_count > 10) {
+        return MESH_ERROR_UNKNOWN;
```

Comment:
If at all more than 10 instances are available in the stack, then the function `rpl_instance_list_read` itself will read only 10 instances. So, automatically it will ignore more than 10 instances. 
I guess we do not need to check this also:
`if (rpl_instance_count > 10) {
     return MESH_ERROR_UNKNOWN;
 }`
What do you suggest ?

Reply:
Im sorry you are absolutely correct no need for the check then you can remove that.

Reply:
Also this is fine as it should not go above 10 in normal case

Reply:
Yes. True. So, I am resolving this comment without modifying anything.